### DISCUSSION TITLE
Update JSON Code for Adobe Korea Page

### DISCUSSION THREAD

**Discussion by seo-sooahlee** [adobe/helix-home#396] [Q&A]
Is there anyone can help with updating the JSON code for Adobe Korea page? We would like to change the Instagram link to a local one 'Adobe Korea Instagram'. This adjustment is for indexing our social media channels on Naver's organic SERP. Thank you.

**Naver's organic SERP:**
[https://search.naver.com/search.naver?sm=tab_hty.top&where=nexearch&ssc=tab.nx.all&query=adobe&oquery=microsoft&tqi=imjbcsqVOsVssbLA2S0sssssslC-328771](url)
<img width="396" alt="MicrosoftTeams-image" src="https://github.com/adobe/helix-home/assets/168574095/d85ef939-630c-41e5-b14c-9d2ed6a838e3">

**Current code:**
<img width="722" alt="MicrosoftTeams-image (1)" src="https://github.com/adobe/helix-home/assets/168574095/f3ea7353-1dff-44e7-b08e-f43505afd67e">

**Updated code:**
https://www.instagram.com/adobekor/

**Comment by MMM3-song** [comment]
@hparra do you think you can suggest anything on this topic?

**Comment by hparra** [comment]
@MMM3-song this is milo functionality. should be possible someone just needs to determine the right way. please make a milo ticket or bring up in #milo-seo channel.
